Sources/UnleashProxyClientSwift/UnleashProxyClientSwift.swift

```diff
@@ -25,15 +25,10 @@ public struct Payload: Codable {
     public let type, value: String
 }
 
-struct Context {
-    let appName: String?
-    let environment: String?
-}
-
 
 @available(macOS 10.15, *)
 public class UnleashClientBase {
-    public var context: [String: String] = [:]
```

Comment:
Correct me if I'm wrong, but won't this be a breaking change? I'm assuming that users will interact with the proxy client, so removing a property might break their setup, right?

Reply:
Not sure why we exposed the internal state before. I'd expect the API to be used through the public methods

Reply:
What you expect and what people actually do are two wildly different things, though 🙈 If this is public, I expect that at least some people use this. However, we're still pre-1.0 and this is might be a fairly small breaking change, so maybe it's okay. Does the swift SDK allow you any other ways to inspect the context? If accessing it directly is the only way to read it, then I don't think we can drop it. Changing the type would still be breaking, but it would be a smaller break.

Reply:
I agree. Keeping the public modifier to keep it visible